server/services/cms.js

```diff
@@ -8,6 +9,22 @@ class CmsService {
     this.#cmsApi = cmsApi;
   }
 
+  async getContent(establishmentName, contentId) {
+    const { type, location } = await this.#cmsApi.lookupContent(
+      establishmentName,
+      contentId,
+    );
+
+    switch (type) {
+      case 'node--page':
+        return this.#cmsApi.get(new BasicPageQuery(location));
+      /// ...other types go here
+      default:
+        // log unsupported type
```

Comment:
when we move all content items into here, we can remove this null check and throw on unexpected types